mods/ra/maps/evacuation/evacuation.lua

```diff
@@ -0,0 +1,365 @@
+		drop  = BadgerDropPoint3.Location,
+	},
+}
+
+AttackGroup = { }
+AttackGroupSize = 5
+SovietInfantry = { "e1", "e2", "e3" }
+SovietVehiclesUpgradeDelay = DateTime.Minutes(4)
+SovietVehicleType = "Normal"
+SovietVehicles =
+{
+	Normal = { "3tnk" },
+	Upgraded = { "3tnk", "v2rl" },
+}
+ProductionInterval =
+{
+	Easy = DateTime.Seconds(10),
+	Normal = DateTime.Seconds(2)
+}
+
+ReinforcementsDelay = DateTime.Minutes(16)
+ReinforcementsUnits = { "2tnk", "2tnk", "2tnk", "2tnk", "2tnk", "2tnk", "1tnk", "1tnk", "jeep", "e1",
+	"e1", "e1", "e1", "e3", "e3", "mcv", "truk", "truk", "truk", "truk", "truk", "truk" }
+
+SpawnAlliedReinforcements = function()
+	if allies2.IsLocalPlayer then
+		UserInterface.SetMissionText("")
+		Media.PlaySpeechNotification(allies2, "AlliedReinforcementsArrived")
+	end
+	Reinforcements.Reinforce(allies2, ReinforcementsUnits, { ReinforcementsEntryPoint.Location, Allies2BasePoint.Location })
+end
+
+Yak = nil
+YakAttack = function(yak)
+	local targets = Map.ActorsInCircle(YakAttackPoint.CenterPosition, WDist.FromCells(10), function(a)
+		return a.Owner == allies1 and not a.IsDead and a ~= Einstein and a ~= Tanya and a ~= Engineer
+	end)
+
+	if (#targets > 0) then
+		yak.Attack(Utils.Random(targets))
+	end
+	yak.Move(Map.ClosestEdgeCell(yak.Location))
+	yak.Destroy()
+	Trigger.OnRemovedFromWorld(Yak, function()
+		Yak = nil
+	end)
+end
+
+SovietTownAttack = function()
+	local units = Utils.Shuffle(Utils.Where(Map.ActorsWithTag("TownAttacker"), function(a) return not a.IsDead end))
+
+	Utils.Do(Utils.Take(5, units), function(unit)
+		unit.AttackMove(TownPoint.Location)
+		Trigger.OnIdle(unit, unit.Hunt)
+	end)
+end
+
+SendParabombs = function()
+	local proxy = Actor.Create("powerproxy.parabombs", false, { Owner = soviets })
+	proxy.SendAirstrikeFrom(BadgerEntryPoint2.Location, ParabombPoint1.Location)
+	proxy.SendAirstrikeFrom(BadgerEntryPoint2.Location + CVec.New(0, 3), ParabombPoint2.Location)
+end
+
+SendParatroopers = function()
+	local proxy1 = Actor.Create("powerproxy.paras1", false, { Owner = soviets })
+	local proxy2 = Actor.Create("powerproxy.paras2", false, { Owner = soviets })
+
+	Paratroopers[1].proxy = proxy1
+	Paratroopers[2].proxy = proxy2
+	Paratroopers[3].proxy = proxy2
+
+	Utils.Do(Paratroopers, function(para)
+		local units = para.proxy.SendParatroopersFrom(para.entry, para.drop)
+		Utils.Do(units, function(unit)
+			Trigger.OnIdle(unit, function(a)
+				if a.IsInWorld then
+					a.Hunt()
+				end
+			end)
+		end)
+	end)
+end
+
+SendAttackGroup = function()
+	if #AttackGroup < AttackGroupSize then
+		return
+	end
+
+	Utils.Do(AttackGroup, function(unit)
+		if not unit.IsDead then
+			unit.Hunt()
+			Trigger.OnIdle(unit, unit.Hunt)
+		end
+	end)
+
+	AttackGroup = { }
+end
+
+ProduceInfantry = function()
+	if SovietBarracks.IsDead or SovietBarracks.Owner ~= soviets then
+		return
+	end
+
+	soviets.Build({ Utils.Random(SovietInfantry) }, function(units)
+		unit = units[1]
```

Comment:
`local unit` please, but I'd drop that entirely and just use `table.insert(AttackGroup, units[1])` instead below.